core/templates/components/question-directives/question-player/question-player.directive.ts

```diff
@@ -194,7 +194,8 @@ angular.module('oppia').directive('questionPlayer', [
               srcset="${getStaticImageUrl('/icons/rocket@2x.webp')}">
               <source type="image/png" 
               srcset="${getStaticImageUrl('/icons/rocket@2x.png')}">
-              <img class="action-button-icon" src="
+              <img alt=""
+              "class="action-button-icon" src="
```

Comment:
Why are the class and src tags quoted together like this? Also, can you align the tags with the alt tag?

Reply:
Done, I think they are generating HTML based on what button is passed into the function. I think it is quoted like this to avoid going over the 80 character limit in the linter.

Reply:
@jimbyo I think we need to remove the extra quote before class.
```suggestion
              class="action-button-icon" src="
```